### PR TITLE
KYP-1914: New plugin version 2.0.0  description

### PR DESCRIPTION
**Pull request by nedick007**

### CODE REVIEW

**Review by velichkoStoev — APPROVED** [review]

File: readme.txt [hunk 1]
```diff
@@ -2,50 +2,51 @@
...
+= Account & Pricing =
+An account is required to use the Metrilo extension. For a free trial, please visit our [sign up page](https://www.metrilo.com/signup?ref=wpplugin). Additional fees apply. Pricing is available on our [pricing page](https://www.metrilo.com/pricing?ref=wpplugin).
 
-* Key sales metrics and marketing performance at a glance
-* Comparison of marketing channels
-* Real-time revenue reporting and many breakdowns
-* Pre-set sales funnel tracking
-* Product performance data to help you develop your brand
+= Features: =
+* Sales performance dashboard - all essential KPIs up-to-date in real time
```
Comment (evgeni-spasov-metrilo) [inline readme.txt]: After the latest changes in our navigation we have a bit different naming for our features. Maybe it's a bit misleading to keep the text as is?
```
text                            navigation

Revenue breakdown               Analytics > Revenue
Product performance             Analytics > Products
Customer Retention              Analytics > Retention
Sales performance dashboard     What would that be in navigation?
```

File: readme.txt [hunk 2]
```diff
@@ -2,50 +2,51 @@
...
-* Recover lost sales
+= Grow With Long Lasting Customer Relationships =
+Metrilo's powerful personalized emails engage better, build relationships and drive repeat sales.
+
+* Send effective data-driven emails
+* Segment your customer base meaningfully and tailor the experience
+* Automate engagement - content drips, replenishment, abandoned carts, etc.
+* Gather feedback automatically - and measure customer satisfaction
+
+Metrilo helps you understand, engage with, and grow your Woocommerce store. The platform unlocks growth through customer retention, which brings marketing costs down and improves ROI. Metrilo is built for brands looking to scale and to build long-lasting customer relationships. We do just that. Getting started only takes a minute and your data is ready to help you grow.
```
Comment (evgeni-spasov-metrilo) [inline readme.txt]: Woocommerce -> Woo**C**ommerce

**Review by evgeni-spasov-metrilo — CHANGES_REQUESTED** [review]

File: readme.txt [hunk 3]
```diff
@@ -13,17 +13,17 @@ Ecommerce Analytics and behaviour-driven customer engagement tools for ecommerce
...
+The extension connects easily to your WooCommerce store and processes sales and marketing data into ready-to-use reports and rich customer insights, including historical data - Metrilo will make past orders and old customers accessible for reactivation, too! Then, you can monitor how your store is doing via the dashboard with all key metrics in real time. The Marketing performance, Revenue breakdown and Product performance reports show opportunities for optimization. The Customer Database makes your first-party customer data usable for segmentation and engagement via emails or retargeting. The Customer Retention feature helps you understand the customer lifecycle and loyalty to drive more repeat sales. And finally, the emails - manual and automated, utilize all data to reach customers at the right time with the right message. Metrilo opens up opportunities to grow your revenue without spending more on marketing.
```
Comment (evgeni-spasov-metrilo) [inline readme.txt]: `The Marketing performance, Revenue and Products reports show opportunities for optimization.`

`The Retention feature helps you understand the customer lifecycle and loyalty to drive more repeat sales.`

File: readme.txt [hunk 4]
```diff
@@ -13,17 +13,17 @@ Ecommerce Analytics and behaviour-driven customer engagement tools for ecommerce
...
-* Sales performance dashboard - all essential KPIs up-to-date in real time
-* Marketing performance reporting and Revenue breakdown - see what brings you sales and is worth investing in
-* Product performance reports - for better product management
+* Metrics Dashboard - all essential KPIs up-to-date in real time
+* Analytics > Revenue - see what brings you sales and is worth investing in
+* Analytics > Products - for better product management
+* Analytics > Retention - monitor the customer lifecycle and loyalty to be able to drive more repeat sales
 * Customer Database - complete profiles of all your customers and numerous filtering options for segmentation
-* Customer Retention - monitor the customer lifecycle and loyalty to be able to drive more repeat sales
 * Email Automation - tailored email engagement on autopilot based on all this data
```
Comment (evgeni-spasov-metrilo) [inline readme.txt]: `Email > Automation`

**Review by evgeni-spasov-metrilo — CHANGES_REQUESTED** [review]

**Review by evgeni-spasov-metrilo — APPROVED** [review]

**Pull request merged**